.github/workflows/wheels.yml

```diff
@@ -9,7 +9,7 @@ env:
   CIBW_SKIP: "cp27-* cp34-* cp35-* *i686 pp* *win32"
 
   # Linux specific build settings
-
+  # TODO - remove eigen

```

Comment:
Maybe worth merging lines 46 and 47 to check that this PR would indeed fix #60. That separation of tests execution was the hotfix for the issue.